lib/commands/run/index.js

```diff
@@ -0,0 +1,152 @@
+const path = require('path');
+const SmapiClient = require('@src/clients/smapi-client');
+const { AbstractCommand } = require('@src/commands/abstract-command');
+const optionModel = require('@src/commands/option-model');
+const AuthorizationController = require('@src/controllers/authorization-controller');
+const ResourcesConfig = require('@src/model/resources-config');
+const CONSTANTS = require('@src/utils/constants');
+const profileHelper = require('@src/utils/profile-helper');
+const jsonView = require('@src/view/json-view');
+const stringUtils = require('@src/utils/string-utils');
+const Messenger = require('@src/view/messenger');
+const CliError = require('@src/exceptions/cli-error');
+const helper = require('./helper');
+
+class RunCommand extends AbstractCommand {
+    name() {
+        return 'run';
+    }
+
+    description() {
+        return 'Starts a local instance of your project as the skill endpoint.'
+            + ' Automatically re-routes development requests and responses between the Alexa service and your local instance.';
+    }
+
+    requiredOptions() {
+        return [];
+    }
+
+    optionalOptions() {
+        return ['debug-port', 'wait-for-attach', 'watch', 'region', 'profile', 'debug'];
+    }
+
+    handle(cmd, cb) {
+        const debugPort = cmd.debugPort || CONSTANTS.RUN.DEFAULT_DEBUG_PORT;
+        const skillCodeRegion = cmd.region || CONSTANTS.ALEXA.REGION.NA;
+        const runRegion = cmd.region || CONSTANTS.ALEXA.REGION.NA;
+        const watch = cmd.watch || false;
+        let skillId, profile;
+
+        try {
+            profile = profileHelper.runtimeProfile(cmd.profile);
+            new ResourcesConfig(path.join(process.cwd(), CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG));
+            skillId = ResourcesConfig.getInstance().getSkillId(profile);
+            if (!stringUtils.isNonBlankString(skillId)) {
+                throw new CliError(`Failed to obtain skill-id for the given profile - ${profile} from project `
+                    + `resource file ${CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG}`);
```

Comment:
this one seems not corrected in this rev - becauase skillid is not in ask-resources, you can just say `Failed to obtain skill-id for the given profile [${profile}], please deploy you skill project first.`